x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_tables.tsx

```diff
@@ -430,9 +464,12 @@ export const RulesTables = React.memo<RulesTableProps>(
               )}
             </HeaderSection>
 
-            {isLoadingAnActionOnRule && !initLoading && (
-              <Loader data-test-subj="loadingPanelAllRulesTable" overlay size="xl" />
-            )}
+            {!initLoading &&
+              (loading || isLoadingRules || isLoadingAnActionOnRule) &&
+              !isRefreshing && (
+                <Loader data-test-subj="loadingPanelAllRulesTable" overlay size="xl" />
```

Comment:
`Loader` doesn't pass `data-test-subj="loadingPanelAllRulesTable"` to its children and renders `data-test-subj="loading-spinner"` internally.

Current Cypress tests use `data-test-subj="loading-spinner"` to identify this loading indicator. It works, but it's somewhat brittle.